config.schema.yaml

```diff
@@ -0,0 +1,40 @@
+---
+credentials:
+  description: "Array of credentials for cloud providers"
+  required: true
+  type: "array"
```

Comment:
That vsphere config makes my head hurt. I'll need to stare at it a bit more to get my head around it